app/controllers/admin/genres_controller.rb

```diff
@@ -0,0 +1,17 @@
+class Admin::GenresController < Admin::BaseController
+
+  def new
```

Comment:
your new method isn't needed at all since the form exists elsewhere (that's all the 'new' action is for, really, just to render the form)